stories/Table.stories.js

```diff
@@ -0,0 +1,627 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Button } from '../src/Button'
+import { Table } from '../src/Table'
+import { TableBody } from '../src/Table/TableBody'
+import { TableHead } from '../src/Table/TableHead'
+import { TableFoot } from '../src/Table/TableFoot'
+import { TableRow } from '../src/Table/TableRow'
+import { TableCellHead } from '../src/Table/TableCellHead'
+import { TableCell } from '../src/Table/TableCell'
+import { TableCellText } from '../src/Table/TableCellText'
+
+const TableFooterButton = () => (
+    <div>
+        <Button primary>Table footer button</Button>
+
+        <style jsx>{`
+            div {
+                text-align: right;
+                padding: 16px 0;
+            }
+        `}</style>
+    </div>
+)
+storiesOf('Table', module)
+    .add('Default', () => (
+        <Table>
+            <TableHead>
+                <TableRow>
+                    <TableCellHead>
+                        <TableCellText label="First name" />
+                    </TableCellHead>
+                    <TableCellHead>
+                        <TableCellText label="Last name" />
+                    </TableCellHead>
+                    <TableCellHead>
+                        <TableCellText label="Last name" />
```

Comment:
Minor nit-pick: all of the column headers are "LastName" instead of their intended values (Incident Date, Last Updated, etc.).  This also exposes the danger of duplicating the Title in multiple places - they will get out of sync.

Reply:
> This also exposes the danger of duplicating the Title in multiple places - they will get out of sync.

True, we could do this dynamically with context as well.. Don't know if that's the right solution here, would be nice if we had these values without having to pass them to every `TableCell`. 

If context is the right thing, I'm wondering about how to change the thead / th components.
It's tempting to just pass an array with strings, but that won't work once one of the head cells needs to have a different rowSpan/colSpan. Extracting it from the children feels wrong..

Reply:
I'm not *sure* this is right, but it is possible.  It also removes the need for an explicit `TableCellHead` component, which if we keep it should probably be a `TableRowHead` with a bunch of `TableCell` children?

Reply:
Maybe title and label are the same thing but you could have anything you'd like...

Reply:
Don't know if this is getting too "centralized"

Reply:
If this table will be used for stuff like the pivot tables, then the thead will have multiple rows.
I think that the `ui-core` component should eventually be used for all apps, so I guess it's not really an option because then things will get very complicated.

@varl any thoughts on this?

Reply:
I mean nested / multi-row headers could also be supported `[ [ 'category', [ 'subCat1', 'subCat2' ] ] ]` (just as an example, maybe with some nice wrapper functions or objects to avoid lots of nested arrays)